server/app/controllers/applicant/ApplicantInformationController.java

```diff
@@ -48,13 +50,15 @@ public ApplicantInformationController(
       FormFactory formFactory,
       ProfileUtils profileUtils,
       ApplicantLayout layout,
-      VersionRepository versionRepository) {
+      VersionRepository versionRepository,
+      ApplicantRoutes applicantRoutes) {
     super(profileUtils, versionRepository);
     this.httpExecutionContext = httpExecutionContext;
     this.messagesApi = messagesApi;
     this.repository = repository;
     this.formFactory = formFactory;
     this.layout = layout;
+    this.applicantRoutes = applicantRoutes;
```

Comment:
**nitpick:** these should all have `checkNotNull`

Reply:
Done